templates/public-adventures/body.html

```diff
@@ -18,6 +17,39 @@
                 >
         </div>
         <div class="flex w-full sm:basis-1/2 gap-2">
+          <div id="adventures" class="flex-1">
```

Comment:
Hi, Annelein! I don't really like the idea of the public adventures being visualized one by one only, and accessible through a filter. If we are removing the tabs (good idea), we should figure out an alternative way to access the adventures. No one is really going to find anything with this UI, because I dont think people will know the adventure's name in advance.

Also I think this change should be made in a sepearate PR, I think this one should tackle just the Hedy page.